### PR TITLE
fix(global): bg and color - default to light mode

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**  
This PR enables light mode by default.

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1477

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Data-table -> WC
3. Check that primary vs secondary is correct.
4. Check the "root" of the HTML.
5. See that it has the correct bg and color based on theme.

### CODE REVIEW

**Review by nathalielindqvist — APPROVED** [review]
LGTM :)

**Comment by EmelieLitwin** [comment]
Not really part of the PR but in most components primary/secondary mode variant variables have been handled like this:
image

E.i. all the background colors etc have been declared at the same place above. Should we do it like that in the data table as well?

**Comment by mJarsater** [comment]
> Not really part of the PR but in most components primary/secondary mode variant variables have been handled like this: image
> 
> E.i. all the background colors etc have been declared at the same place above. Should we do it like that in the data table as well?

Yes, I think that would be a good approach 👍

File: tegel/src/global/global.scss [hunk 1]
```diff
@@ -86,17 +86,16 @@ body {
   flex: 1;
 }
 
-html,
 .sdds-mode-light {
```
Comment (alexandereneroth) [inline tegel/src/global/global.scss:89]:
```suggestion
:root,
.sdds-mode-light {
```

**Review by alexandereneroth — CHANGES_REQUESTED** [review]
I think the issue is rather that the background style is set on the :host of the webcomponent (&lt;sdds-table-body-row>), I think we should not style :host as styles put on :host are outside the shadow dom and are not protected (as this issue demonstrates).
It would be better if we put the background on an inner element inside :host instead.

**Pull request merged**